popup.js

```diff
@@ -0,0 +1,259 @@
+
+articlesContainer.innerHTML = `
+<section class="articles" id="portfolio">
+
+
+<h2 class="${projects[0].class1}">${projects[0].title1}</h2>
+<hr id="first-hr">
+
+<img id="GymFit" src="${projects[0].img}" alt="GymFit">
+
+<article class="${projects[0].class}" id="${projects[0].id}">
+<h2 id="${projects[0].idh2}">${projects[0].title}</h2>
+<p>${projects[0].description}</p>
+<ul class="boxes white">
+<li>${projects[0].technologies[0]}</li>
+<li>${projects[0].technologies[1]}</li>
+<li>${projects[0].technologies[2]}</li>
+<li>${projects[0].technologies[3]}</li>
+</ul>
+<button class="button" id="project1">See Project</button>
+</article>
+
+
+
+<article class="${projects[1].class}" id="${projects[1].id}">
+<h2>${projects[1].title}</h2>
+<p>${projects[1].description}</p>
+<ul class="boxes black">
+<li>${projects[1].technologies[0]}</li>
+<li>${projects[1].technologies[1]}</li>
+<li>${projects[1].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+
+<article class="${projects[2].class}" id="${projects[2].id}">
+<h2 class="mobile">${projects[2].title}</h2>
+<h2 class="desktop">${projects[2].title2}</h2>
+<p>${projects[2].description}</p>
+<ul class="boxes black">
+<li>${projects[2].technologies[0]}</li>
+<li>${projects[2].technologies[1]}</li>
+<li>${projects[2].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+
+<article class="${projects[3].class}" id="${projects[3].id}">
+<h2 class="mobile">${projects[3].title}</h2>
+<h2 class="desktop">${projects[3].title2} </h2>
+<p>${projects[3].description}</p>
+<ul class="boxes black">
+<li>${projects[3].technologies[0]}</li>
+<li>${projects[3].technologies[1]}</li>
+<li>${projects[3].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+
+<article class="${projects[4].class}" id="${projects[4].id}">
+<h2>${projects[4].title}</h2>
+<p>${projects[4].description}</p>
+<ul class="boxes black">
+<li>${projects[4].technologies[0]}</li>
+<li>${projects[4].technologies[1]}</li>
+<li>${projects[4].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+
+<article class="${projects[5].class}" id="${projects[5].id}">
+<h2 class="mobile">${projects[5].title}</h2>
+<h2 class="desktop">${projects[5].title2}</h2>
+<p>${projects[5].description}</p>
+<ul class="boxes black">
+<li>${projects[5].technologies[0]}</li>
+<li>${projects[5].technologies[1]}</li>
+<li>${projects[5].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+
+<article class="${projects[6].class}" id="${projects[6].id}">
+<h2 class="mobile">${projects[6].title}</h2>
+<h2 class="desktop">${projects[6].title2} </h2>
+<p>${projects[6].description}</p>
+<ul class="boxes black">
+<li>${projects[6].technologies[0]}</li>
+<li>${projects[6].technologies[1]}</li>
+<li>${projects[6].technologies[2]}</li>
+</ul>
+<button class="button">See Project</button>
+</article>
+</section>
```

Comment:
There is a huge amount of code(91 lines) that repeats itself on these lines, unfortunately. One of the key aspects as developers is to maintain [DRY](https://codinglead.co/javascript/what-is-DRY-code) code for maintainability reasons. _'One copy is much easier to maintain than multiple copies'_.

Using a loop to create your projects section dynamically would benefit you a lot in this case! 
Kindly make the necessary changes to meet the project requirements and make our code rock 🔥